packages/RichText-Core.package/RTEAddComponent.class/instance/exportMarkdown.st

```diff
@@ -1,6 +1,4 @@
 api
 exportMarkdown
 
-	(self coreObject notNil)
-		ifTrue: [^ self coreObject exportMarkdown]
-		ifFalse: [^ '']
+	^ self coreObject exportMarkdown
```

Comment:
I think you made something wrong when merging. exportMarkdown should never return "nil" instead return an empty string. This was a fix and you remove it here. We should consider writing a test for this edge case. Could you do this?
It's just ... at the AddComponent Tests
```smalltalk
testExportMarkdownWhenNoComponentChosen

    self assert: component exportMarkdown equals: ''
```